config/routes.rb

```diff
@@ -62,6 +62,7 @@
       get :no_data_allowed
       get :new_from_invite_data
     end
+    get 'autocompletes/:field_name/', to: 'autocompletes#index', as: 'autocomplete'
```

Comment:
Hum, I'm not sure whether we should pass the `institution_id` in the URL. The whole project doesn't work that way, but instead passes the `context` param (institution + all sites or a single site) :thinking:

I'm wondering what we actually want, here. Shall we search within the whole institution (better to avoid duplicates at the global institution level)? Or shall we respect the currently selected site for autocompletes?

Poke @leandroradusky